### ISSUE TITLE
[TO3] Hiperparameter Tuning

### ISSUE THREAD

**Issue by birrulwldain** [birrulwldain/skripsi-21#16]
Optimasi unit LSTM, dropout rate, learning rate.

**Acceptance Criteria:**
- Model dengan val_loss terbaik terseleksi.
